src/machine.rs

```diff
@@ -279,12 +282,13 @@ impl<'tcx> Evaluator<'tcx> {
             panic_payload: None,
             time_anchor: Instant::now(),
             layouts,
+            threads: Default::default(),
```

Comment:
```suggestion
            threads: ThreadManager::default(),
```
just to match our usual style